lib/dish.rb

```diff
@@ -0,0 +1,16 @@
+class Dish
+
+  attr_reader :dish, :price
```

Comment:
Yaara suggested that it's better to do these as defined methods in the class, although tbh I much prefer attr_reader too hahaha